python/lsst/pipe/tasks/warpAndPsfMatch.py

```diff
@@ -38,6 +41,11 @@ class WarpAndPsfMatchConfig(pexConfig.Config):
         dtype=afwMath.Warper.ConfigClass,
         doc="warper configuration",
     )
+    reverseWarpAndMatch = pexConfig.Field(
```

Comment:
It occurs to me that `reverseWarpAndMatch` only makes sense if you know what the nominal order of warp and match is.  Perhaps something like `matchFirst` or `matchThenWarp` would be more clear?